production/helm/loki/templates/backend/statefulset-backend.yaml

```
@@ -213,7 +213,7 @@ spec:
       {{- end }}
       {{- with .Values.backend.dnsConfig }}
       dnsConfig:
-        {{- tpl . $ | nindent 8 }}
+        {{- toYaml . | nindent 8 }}
```

This should prevent a breaking change:

```suggestion
        {{- if kindIs "map" . }}
        {{- tpl (toYaml .) $ | nindent 8 }}
        {{- else }}
        {{- tpl . $ | nindent 8 }}
        {{- end }}
```